.github/workflows/test-install-scripts.yml

```diff
@@ -141,7 +141,7 @@ jobs:
     - name: Prepare Rocky
       if: ${{ matrix.distribution.type == 'rocky' }}
       run: |
-        yum install --assumeyes git sudo
+        yum install --assumeyes git sudo python39
```

Comment:
I see that `python3` gets you 3.6...  So that doesn't work.  Should we pick 3.8 as our oldest supported version?

Reply:
I picked 3.9 just so I wouldn't have to come back in a few months and remove 3.8 when it's deprecated...

Reply:
The nominal search we already should do ought to catch this.  `3\.?8` or somesuch.